packages/workspace/src/generators/new/schema.json

```diff
@@ -56,7 +56,7 @@
     "packageManager": {
       "description": "The package manager used to install dependencies.",
       "type": "string",
-      "enum": ["npm", "yarn", "pnpm"]
+      "enum": ["npm", "yarn", "pnpm", "bun"]
```

Comment:
Nice spot! Cent believe I missed that.

Reply:
All good. I tested it out locally and it's working now. Checking a few more things.